src/blockbandedcholesky.jl

```diff
@@ -0,0 +1,91 @@
+
+
+sparsecholesky(A::Symmetric{<:Real,<:BlockArray},
+::Val{false}=Val(false); check::Bool = true) = sparsecholesky!(cholcopy(A); check = check)
+
+
+
+function _blockbandedcholesky!(A::BlockArray{T}, ::Type{UpperTriangular}) where T<:Real
```

Comment:
I don't understand why this is a separate function, not just `block_chol!` from beliw.

Reply:
The only difference is that I added a check of trivial blocks from backwards. It actually do the same thing on dense matrices. We can have another parameter for activating the 'check'.